user_journey/views/funnel_analysis/client_aggregates.view.lkml

```diff
@@ -1,6 +1,6 @@
 include: "//looker-hub/firefox_desktop/views/clients_last_seen.view.lkml"
 
-view: client_properties {
+view: client_aggregates {
```

Comment:
Is `client_aggregates` a better name here?

Reply:
Hmm, maybe `browser_properties`? I think `client_properties` made more sense than `client_aggregates`.